src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -61,10 +62,18 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import javax.annotation.Nullable;
+import org.tikv.common.util.HistogramUtils;
 
 /** Implementation of {@link ClientCall}. */
 final class ClientCallImpl<ReqT, RespT> extends ClientCall<ReqT, RespT> {
 
+  public static final Histogram perfmarkClientCallImplDuration =
+      HistogramUtils.buildDuration()
+          .name("perfmark_client_call_impl_duration_seconds")
+          .help("Perfmark client call impl duration seconds")
```

Comment:
We should use customized buckets to better observe small durations.

Reply:
HistogramUtils is already used our customized duration buckets.

Reply:
@sunxiaoguang PTAL

Reply:
Sorry, I overlooked that.